pt-BR/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -92,27 +92,27 @@
 <!ENTITY findPrevCmd.label "Localizar anterior">
 <!ENTITY findPrevCmd.key "g">
 <!ENTITY findPrevCmd.key2 "VK_F3">
-<!ENTITY findPrevCmd.accesskey "a"> 
+<!ENTITY findPrevCmd.accesskey "a">
 
 <!-- View Menu -->
-<!ENTITY viewMenu.label "Exibir"> 
+<!ENTITY viewMenu.label "Exibir">
 <!ENTITY viewMenu.accesskey "x">
 <!ENTITY customizeToolbar.label "Personalizar…">
 <!ENTITY customizeToolbar.accesskey "P">
-<!ENTITY showFormatToolbarCmd.label "Formatting Toolbar">
+<!ENTITY showFormatToolbarCmd.label "Barra de formatação">
 <!ENTITY showFormatToolbarCmd.accesskey "F">
-<!ENTITY pbShowCcField.label "Cc Address Field">
+<!ENTITY pbShowCcField.label "Campo de endereço Cc">
 <!ENTITY pbShowCcField.accesskey "C">
 <!ENTITY pbShowCcField.key "c">
-<!ENTITY pbShowBccField.label "Bcc Address Field">
+<!ENTITY pbShowBccField.label "Campo de endereço Cco">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbShowBccField.label "Campo de endereço Bcc">
@brunotiussi please don't translate the string Bcc.